### ISSUE TITLE
NTR: Anti-soluble liver antigen/liver pancreas (anti-SLA/LP) (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8778]
New term request Anti-soluble liver antigen/liver pancreas (anti-SLA/LP) (chang6_190715193852):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000186
- label: Anti-soluble liver antigen/liver pancreas antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:19302185
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against soluble liver antigen.
- comment: Antibodies against SLA/LP are now considered as being highly specificmarkers for auto-immune hepatitis type 1.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8778
- synonyms: Anti-SLA/LP(NONE)
